src/modules/dashboard/data/queries/colonies.ts

```diff
@@ -492,14 +492,20 @@ export const getColonyTokenBalance: Query<
     { colonyManager }: { colonyManager: ColonyManager },
     { colonyAddress },
   ) => colonyManager.getColonyClient(colonyAddress),
-  async execute(colonyClient, { tokenAddress: token }) {
-    const {
-      total: nonRewardsPotsTotal,
-    } = await colonyClient.getNonRewardPotsTotal.call({ token });
+  async execute(colonyClient, { domainId, tokenAddress: token }) {
+    const { potId } = await colonyClient.getDomain.call({ domainId });
     const {
       balance: rewardsPotTotal,
-    } = await colonyClient.getFundingPotBalance.call({ potId: 0, token });
-    return new BigNumber(nonRewardsPotsTotal.add(rewardsPotTotal).toString(10));
+    } = await colonyClient.getFundingPotBalance.call({ potId, token });
+    if (domainId === 0) {
+      const {
+        total: nonRewardsPotsTotal,
+      } = await colonyClient.getNonRewardPotsTotal.call({ token });
+      return new BigNumber(
+        nonRewardsPotsTotal.add(rewardsPotTotal).toString(10),
```

Comment:
I think you can avoid converting these to string, BN constructor should take a BN.